invokeai/backend/model_manager/metadata/fetch/civitai.py

```diff
@@ -164,6 +164,7 @@ def _from_model_json(self, model_json: Dict[str, Any], version_id: Optional[int]
                 AllowDerivatives=model_json["allowDerivatives"],
                 AllowDifferentLicense=model_json["allowDifferentLicense"],
             ),
+            trigger_phrases=version_json["trainedWords"],
```

Comment:
This is already on the metadata object as `trained_words`

Reply:
Yeah I was trying to not use the civit key just to spite them

Reply:
I mean the metadata schema already extracts it a few lines earlier

Reply:
Do you prefer `trained_words`? I don't feel strongly

Reply:
`trigger_phrases` works for me, and is more consistent with the terminology we use for textual inversion embeddings.